### ISSUE TITLE
Fix code scanning alert - Depending upon JCenter/Bintray as an artifact repository

### ISSUE THREAD

**Issue by JonasCir** [hzi-braunschweig/SORMAS-Project#7513]
Tracking issue for:
- [ ] https://github.com/hzi-braunschweig/SORMAS-Project/security/code-scanning/7

**Comment by JonasCir** [comment]
Created from the details view of the alert, quite nice user flow! :)

**Comment by JonasCir** [comment]
Blocked on https://github.com/hzi-braunschweig/SORMAS-Project/issues/7872

**Comment by JonasCir** [comment]
Just a duplicate of  #5065
